tests/e2e/account/pw_register_test.py

```diff
@@ -0,0 +1,215 @@
+"""Test account registration."""
+# pylint: disable=redefined-outer-name
+
+import re
+
+import pytest
+from playwright.sync_api import Page, expect
+from pytest_bdd import given, parsers, scenarios, then, when
+
+scenarios('account/register.feature')
+pytestmark = pytest.mark.usefixtures('live_server')
+
+TWOFLOWER_EMAIL = 'twoflower@example.org'
+TWOFLOWER_PHONE = '+12015550123'
+TWOFLOWER_PASSWORD = 'te@pwd3289'  # nosec
+
+pytestmark = pytest.mark.filterwarnings(
+    "ignore:Object of type <AccountPhone> not in session",
+    "ignore:Object of type <AccountEmail> not in session",
```

Comment:
Don't filter these. The problem is in the `add` classmethod of EmailAddress and PhoneNumber. It should be resolved there.